ui/src/components/member/MemberRow.js

```diff
@@ -169,9 +177,21 @@ export default class MemberRow extends React.Component {
         let clickDelete = this.onClickDelete.bind(this, this.state.deleteName);
         let submitDelete = this.onSubmitDelete.bind(this, this.props.domain);
         let clickDeleteCancel = this.onClickDeleteCancel.bind(this);
-
+        let isSuccess =
+            member.memberName +
+                '-' +
+                this.props.category +
+                '-' +
+                this.props.domain +
+                '-' +
+                this.props.collection ===
```

Comment:
won't this return true instead of the collection/newMember?

Reply:
yup this is the flag whether to show the animation for a specific row